Frontend/src/Components/BatteryPack/BatteryPack.js

```diff
@@ -0,0 +1,150 @@
+                            lineHeight={1}
+                            color={fanEnableTextColor}
+                        >
+                            {fanEnableText}
+                        </Text>
+                    </Center>
+                    <Text fontSize='sm' lineHeight='2em' marginLeft='2.5%'>Fan Speed:</Text>
+                    <Spacer />
+                    <Text fontSize='sm' paddingRight='0.5em' lineHeight='2em'>
+                        {props.data?.fan_speed[0] ?? -1}
+                    </Text>
+
+                    <RangeBar 
+                            w='3em'
+                            h='1em'
+                            borderRadius='0px'
+
+                            val={(props.data?.fan_speed[0] ?? -1.0)}
+                            min={fanSpeedMin}
+                            max={CONSTANTS.fan_speed.MAX}
+                    />
+                </Flex>
+            </Center>
+            <RangeRow
+                dataTitle="Pack Current"
+                dataValue={props.data?.pack_current[0] ?? -1.0}
+                dataConstant={CONSTANTS.pack_current}
+
+                bg={headerBg}
+            />
+            <RangeRow
+                dataTitle="Pack Voltage"
+                dataValue={props.data?.pack_voltage[0] ?? -1.0}
+                dataConstant={CONSTANTS.pack_voltage}
+            />
+            <RangeRow
+                dataTitle="Pack Power Out"
+                dataValue={props.data?.pack_power[0] ?? -1.0}
+                dataConstant={CONSTANTS.pack_power}
+
+                bg={headerBg}
+            />
+            <RangeRow
+                dataTitle="Pack SoC"
+                dataValue={props.data?.soc[0] ?? -1.0}
+                dataConstant={CONSTANTS.soc}
+            />
+            <RangeRow
+                dataTitle="Pack Resistance"
+                dataValue={props.data?.pack_resistance[0] ?? -1.0}
+                dataConstant={CONSTANTS.pack_resistance}
+
+                bg={headerBg}
+            />
+            <RangeRow
+                dataTitle="Adapt. Total Capacity"
+                dataValue={props.data?.adaptive_total_capacity[0] ?? -1.0}
+                dataConstant={CONSTANTS.adaptive_total_capacity}
+            />
+            <Center
+                borderTopWidth={1}
+                borderRightWidth={1}
+                borderColor={borderColor}
+                bg={((props.data?.charge_enable[0] ?? -1.0) != CONSTANTS.charge_enable.MIN && (props.data?.charge_enable[0] ?? -1.0) != CONSTANTS.charge_enable.MAX) ?
+                    getColor("errorBg", colorMode) : headerBg}
+                lineHeight='1.2em'
+                height='100%'
+            >
+                <Flex w='95%' alignItems='center'>
+                    <Text fontSize='sm' lineHeight='2em'>Charge Enable:</Text>
+                    <Spacer />
+                    <Center
+                        border='2px'
+                        w='3em'
```

Comment:
```suggestion
                        w='3.5em'
```
If 3.5em doesn't cause issues on your end, I think it would be worth using that. 3 is a bit too small on my end

Reply:
Fixed